### ISSUE TITLE
Mutiply defined label in LaTeX output

### ISSUE THREAD

**Issue by Karmaki** [sphinx-doc/sphinx#5325]
I am migrating an existing document that used to be compiled with an old version of sphinx (1.3.6) to a newer version (1.7.7 installed through pip) and I have some problems with the labels in the LaTeX output when I have several labels for the same section title.

Example:
```
.. _opt-fct:
.. _opt-prop:

How to Run
-------------
```

gives in the LaTeX file:
```
\phantomsection\label{\detokenize{man/tuto:opt-fct}}

\paragraph{How to Run}
\label{\detokenize{man/tuto:how-to-run}}\label{\detokenize{man/tuto:opt-prop}}\label{\detokenize{man/tuto:opt-fct}}
```

Then, when the document is compiled, I get:
```
LaTeX Warning: Label `ref/options:opt-fct' multiply defined.
```

I would be very happy to find at least a workaround since I have many of those in my document.

**Comment by Karmaki** [comment]
The problem already exists in the 1.6.7 version (that comes with the last LST Ubuntu 18.04).

I forgot to say that the problem can be easily reproduce in a fresh `sphinx-quickstart` installation. Just copy/paste the lines above in the `index.rst` file and run `make latexpdf`.

**Comment by Karmaki** [comment]
A workaround is to keep only one label before the section title, and to put the others between the title and the body of the section.

**Comment by tk0miya** [comment]
Fixed by #5332.
Thank you for reporting.
